caffeine/build.gradle

```diff
@@ -5,7 +5,6 @@ android {
     buildToolsVersion "20.0.0"
```

Comment:
Let's bump up `compileSdkVersion`, `buildToolsVersion`, and `targetSdkVersion` too.